apps/docs/docs/getting-started/01-development.md

```diff
@@ -7,17 +7,42 @@ description: Getting started - development
 
 ## Using Docker compose
 
-The fastest way to get up and running is by using docker. Running `pnpm start:dev` will start the project using the `docker-compose.dev` file this configuration also includes the database. If everything goes correctly nothing else will be required to do and you can visit the following urls.
+The fastest way to get up and running is by using docker. Running `yarn start:docker` will start the project using the `docker-compose` file this configuration also includes the database. If everything goes correctly nothing else will be required to do and you can visit the following urls.
 
-- [http://localhost](http://localhost) or localhost:9000 for the web application
-- [http://cms.localhost](http://cms.localhost) or localhost:9001 for the cms
-- [http://docs.localhost](http://docs.localhost) or localhost:9002 for the documentation
+- [http://localhost](http://localhost) for the web application
+- [http://localhost/cms](http://localhost/cms) for the CMS (Strapi)
+- [http://localhost/docs](http://localhost/docs) for the documentation (Docsaurus)
+
+The reverse proxy runs on port 80. By default, no other services are exposed to the host directly. See below on how to override this behavior and others.
+
+### Overriding User and Group
+
+On Linux / Mac, you may need to override the user and group that the containers run as. To do this, copy `.env.example` to `.env` if you haven't already, and modify `UID` and `GID` respectively to your user and/or group ID. You can find these by running `id` in your terminal.
+
+### Overriding Docker
+
+If you want to override the Docker configuration, you can create a `docker-compose.override.yml` file. This file will be used by docker-compose to override the default configuration. The syntax and options are the same as the default `docker-compose.yml` file. More information can be found in the [docker-compose documentation](https://docs.docker.com/compose/extends/).
```

Comment:
Just wondering, what could be a reason for that override?!

Reply:
Several reasons;

When I spoke to @onursagir, he said that while he agrees with me that Doppler shouldn't be in the docker file, he still wants to keep the project compatible with it. 

The way to achieve that compatibility going forward is by chaining docker-compose and `doppler run` like so - `doppler run -- docker-compose up -d`.

Now imagine Doppler is injecting a variable call `SECRET_VALUE` to `web`. By using `doppler run` you passed the environment to the docker-compose context, but not to the container itself. In order to cascade `SECRET_VALUE` to the `web` container, you'd have to specify it in the `docker-compose.yml` like so;
```yaml
web:
  environment:
    - SECRET_VALUE
```

But, since the file is versioned, I offer the git ignored `docker-compose.override.yml` instead. 

Beyond Doppler, it could become relevant for any configuration change a local user would want to make when running with Docker.